specs/language/statements.tex

```diff
@@ -1,2 +1,93 @@
 \Ch{Statements}{Stmt}
 \Sec{Label Statements}{Stmt.Label}
+\Sec{Attributes}{Stmt.Attr}
+\Sub{Unroll Attribute}{Stmt.Attr.Unroll}
+The \texttt{[unroll]} and \texttt{[unroll(n)]} attribute qualifiers can be used
+ to specify that a loop (\texttt{for}, \texttt{while} and \texttt{do while} 
+loops) can be unrolled. This attribute qualifier can be used to specify full 
```

Comment:
I would probably specify that the _unroll_ attribute is only valid when applied to _iteration-statements_, then you don't need to explicitly list the loop keywords.